apps/src/dance/Dance.js

```diff
@@ -447,6 +455,10 @@ Dance.prototype.playSong = function (url, callback, onEnded) {
   });
 };
 
+Dance.prototype.playSound = function (soundName, options) {
+  this.studioApp_.playAudio(soundName, options);
```

Comment:
Could we use the `Sounds` APIs directly to avoid increased dependency on `StudioApp`? Mostly from the perspective of reusing `DanceAiModal` in the future.